types/node/tls.d.ts

```diff
@@ -452,7 +452,7 @@ declare module 'tls' {
          * SecureContext.) If SNICallback wasn't provided the default callback
          * with high-level API will be used (see below).
          */
-        SNICallback?: ((servername: string, cb: (err: Error | null, ctx: SecureContext) => void) => void) | undefined;
+        SNICallback?: ((servername: string, cb: (err: Error | null, ctx: SecureContext | undefined) => void) => void) | undefined;
```

Comment:
You should also be able to skip the argument entirely, right?

```suggestion
        SNICallback?: ((servername: string, cb: (err: Error | null, ctx?: SecureContext) => void) => void) | undefined;
```

Reply:
You could probably skip both arguments, but I'm not sure if this was intended.

https://github.com/nodejs/node/blob/a1fac5b9a5979b049446c05ab961819fda843e29/lib/_tls_wrap.js#L215-L231

Reply:
I've changed as requested.